dockernet/scripts/community-pool-staking/stake_proposal_legacy.sh

```diff
@@ -0,0 +1,43 @@
+#!/bin/bash
+set -eu 
+SCRIPT_DIR=$( cd -- "$( dirname -- "${BASH_SOURCE[0]}" )" &> /dev/null && pwd )
+source ${SCRIPT_DIR}/../../config.sh
+
+deposit_ica_account=$(GET_ICA_ADDR GAIA community_pool_deposit)
```

Comment:
You are testing this on GAIA because dydx no longer has the legacy gov?

Reply:
yup exactly